Sources/Preferences/util.swift

```diff
@@ -47,3 +47,16 @@ extension NSView {
 		return result
 	}
 }
+
+extension Bundle {
+	var appName: String? {
```

Comment:
Why `appName` and not `name`? A bundle could also be a framework or something else.

Reply:
Is it really worth making this an optional? I was thinking having it also fall back to bundle ID and then `<Unknown>`, so it's guaranteed to return something. I try to not have optionals when not necessary. Not sure if my thinking make sense though?...

Reply:
Hmm, how about we keep this as is and also add a `displayName` property that is guaranteed to display something (like outlined above)?

Reply:
> Why `appName` and not `name`? A bundle could also be a framework or something else.

Limiting the use to apps via the name was intentional (because I narrowed this down too much was in part why I didn't think of including `"CFBundleName"` at first): For what other kind of reference/main bundle would you want to have a preference pane? It can only be an app in the end. And we want to display an app name as per the HIG, not just any string or identifier. The name of an app is visible to users, the name of a framework most likely isn't and matters to developers foremost. Conceptually, this is meant to get you the app name, not just any generic bundle name, and I wanted to capture this intent.

I'm torn on the `Optional` thing. Not having to unwrap the optional value is nice in principle, but it is only used in 1 place, and I wonder if it currently better shows the point of "failure", providing an answer to the question "why is the window title showing `<Unknown Name>`". At least on GitHub. In Xcode, you can Cmd-Ctrl-Click right through to the definition, so there's not much of a real problem. Ah well, I think I'll go with providing a fallback value in `appName` directly. There's no other calling code anyway that makes using 2 properties more useful at this moment.

Reply:
Yeak, makes sense. I'm just over-thinking this.